core-services/release-controller/_releases/priv/release-ocp-4.16.json

```diff
@@ -138,6 +138,20 @@
                 "name": "periodic-ci-openshift-release-master-nightly-4.16-e2e-aws-ovn-single-node-serial-priv"
             }
         },
+        "aws-ovn-single-node-techpreview": {
```

Comment:
I had to add those `priv` entries, where are they used?

Reply:
@bradmwilliams Would know for sure, but I think it had to do with CVE validation in the private environments by adding the `-priv` suffix.

Reply:
You should only ever modify the files in: `core-services/release-controller/_releases` and then run a `make release-controllers` to automatically generate the files in: `core-services/release-controller/_releases/priv`.

Reply:
Yes, I've run `make release-controllers` as requested [in the error message](https://github.com/openshift/release/blob/68d653bcd1431520f2121f4446ce81b6c3b7e056/hack/validate-release-jobs.py#L106C1-L106C121)
I just wanted to know more about those `priv` jobs.